### DISCUSSION TITLE
Broken link in Markup Reference->Code Blocks

### DISCUSSION THREAD

**Discussion by ivergara** [pradyunsg/furo#591] [Potential Issue] [answered]
In the [documentation](https://pradyunsg.me/furo/reference/code-blocks/) concerning code blocks, the link at the end pointing to https://myst-parser.readthedocs.io/en/latest/using/intro.html#block-level-directives-with-myst-markdown is broken as in that anchor doesn't exist.

**Comment by pradyunsg** [comment, marked as answer]
Nice catch!

I've filed #593 for this. A PR would be welcome!
